### ISSUE TITLE
Every few days phone gets no service, need to reboot

### ISSUE THREAD

**Issue by VanillaEssence** [Sticktron/g0blin#304]
Hey when I'm jailbroken every few days my phone will stop connecting to cell network and have to reboot my device thereby losing my jailbreak.

iPhone 6s 10.3.1
Australia

**Comment by OrkhanAlikhanov** [comment]
Did you try switching airplane mode on and off? As a quick workaround

**Comment by VanillaEssence** [comment]
Yes, I've tried every variation of trying to switch off mobile. Only fix is when I full reboot the device which is really annoying. Basically means the jailbreak is unusable for me atm

**Comment by Obsession-Alvin** [comment]
你之前是否使用M 越狱手机
